src/server/common.h

```diff
@@ -383,4 +383,32 @@ struct BorrowedInterpreter {
 
 extern size_t serialization_max_chunk_size;
```

Comment:
why serialization_max_chunk_size is global? can we just use the flag value in snapshot.cc?

Reply:
The reason is global is that we use it/cache the flag value in `ThreadLocalMutex`. 

1. When big value serialization is off, locking/unlocking the mutex is `noop`
2. When big value ser is on, locking/unlocking also checks that the same fiber does not lock the same mutex twice (it's for to detect deadlocks)